Makefile

```diff
@@ -159,6 +159,7 @@ CONTROLLER_TOOLS_VERSION ?= v0.9.2
 CRDOC_VERSION ?= v0.5.2
 OPERATOR_SDK_VERSION ?= 1.23.0
 ENVTEST_VERSION ?= latest
+CERTMANAGER_VERSION ?= 1.8.0
```

Comment:
I think certmanager 1.9.1 is out.  Is there any reason we aren't going with the latest version?

Reply:
nope, we can upgrade to the latest.